src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -11,6 +11,8 @@ export type MentionSuggestionsPopoverProps = Pick<
   PopoverProps,
   'open' | 'onClose' | 'anchorElementRef'
 > & {
+  /** Whether the list of users is currently being loaded */
+  loadingUsers: boolean;
```

Comment:
Other components up the tree are handling the loading status via the same prop that holds the list of users, but in here, I settled on a separate prop, because the parent component (`MarkdownEditor`) applies some logic to the list of users, including checking its length in some cases, or extracting items from it.

If the list of users becomes `UserItem[] | 'loading'`, there are a number of places where we suddenly need to check if the value is an array, in addition to the other existing checks.

Because of that, the `userSuggestions` list is kept always as an array, inferred from `usersForMentions`, and we only pass `loadingUsers={usersForMentions === 'loading'}` in one place.